EssentialFeed/EssentialFeediOSTests/Image Comments UI/ImageCommentsSnapshotTests.swift

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeediOS
+@testable import EssentialFeed
```

Comment:
Does this import need to be `@testable`? If not, please remove it.
```suggestion
import EssentialFeed
```

Reply:
No, it doesn't. Removed it.